tasks/e2e-installs.sh

```diff
@@ -162,6 +162,8 @@ exists tsconfig.json
 exists src/react-app-env.d.ts
 checkTypeScriptDependencies
 
+node node_modules/react-scripts/scripts/utils/verifyTypeScriptSetup.js
```

Comment:
What is this needed for?

Reply:
This should be removed, I believe I solved it properly with a change I made to `react-scripts/init.js`.